README.md

```diff
@@ -1,6 +1,6 @@
-# GitHub Action for Code Dx
+# GitHub Action for SRM
 
-This GitHub action can be used to push source code, binaries, and scan results to a [Code Dx](https://codedx.com) instance from within a GitHub workflow; source and binaries are automatically scanned by Code Dx using its built-in analysis tools.
+This GitHub action can be used to push source code, binaries, and scan results to an [SRM](https://codedx.com) instance from within a GitHub workflow; source and binaries are automatically scanned by SRM using its built-in analysis tools.
```

Comment:
Link should be updated - https://www.synopsys.com/software-integrity/software-risk-manager.html